test/test_excerpt.rb

```diff
@@ -102,11 +104,12 @@ def do_render(document)
         setup do
           @rendered_post = @post.dup
           do_render(@rendered_post)
-          @extracted_excerpt = @rendered_post.data['excerpt']
+          @extracted_excerpt = @rendered_post.data["excerpt"]
         end
 
         should "be the first paragraph of the page" do
-          assert_equal "<p>First paragraph with <a href=\"http://www.jekyllrb.com/\">link ref</a>.</p>\n\n", @extracted_excerpt.output
+          assert_equal "<p>First paragraph with <a href=\"http://www.jekyllrb.com/\">"\
+                       "link ref</a>.</p>\n\n", @extracted_excerpt.output
```

Comment:
Same as above – please put the string back onto one line and use some temporary variable like `expected` to hold it.